bootstrap/bacardi.sh

```diff
@@ -40,7 +40,10 @@ if [ ! -f .last_update ] || [ package.json -nt .last_update ]; then
   npm install
 fi
 
+if [ "$1" ]; then
```

Comment:
IMHO, if there is no subcommand, can we return early?

Reply:
Do you mean not going through default task in gulp?